slices/git.yaml

```diff
@@ -0,0 +1,34 @@
+package: git
+
+essential:
+  - git_copyright
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
+      - libpcre2-8-0_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/git:
+      /usr/bin/git-shell:
+      /usr/bin/scalar:
+
+  core:
+    essential:
+      - git_bins
+      - libcurl3t64-gnutls_libs
+      - libexpat1_libs
+    contents:
+      /usr/lib/git-core/**:
```

Comment:
I wonder if we should merge the `bins` and `core` slices into one: `bins`. Since together they are offering binaries and executables. Let me know what you think!

Reply:
I would keep both, there are a few extra bins in core which are not needed for simple tasks. Infact, it might be nice to include a slice that only include the single git binary (/usr/bin/git). We could call it minimal for instance. The use case here would be build envs that need to extract details about the repo.

Reply:
Alright, sounds good.